.travis.yml

```diff
@@ -13,9 +13,15 @@ python:
   - "2.7"
 
 env:
-  - VERSION="8.0" LINT_CHECK="1"
-  - VERSION="8.0" ODOO_REPO="odoo/odoo" LINT_CHECK="0"
-  - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
+  global:
```

Comment:
@Hayyan-Ebrahem Analysing this will go as follows:
1. Look at the [most recent version of this file on the 8.0 branch in the original OCA repo](https://github.com/OCA/l10n-netherlands/blob/8.0/.travis.yml), and activate the [Blame view](https://github.com/OCA/l10n-netherlands/blame/8.0/.travis.yml).
2. See that this change was [added 9 months ago by Holger in this commit](https://github.com/OCA/l10n-netherlands/commit/e0b7bfb5aa9721b31f247a054577d340db8ca07b)
3. Conclude that this should not be in this PR.
4. Look at the [commits in this PR](https://github.com/sunflowerit/l10n-netherlands/pull/1/commits) and conclude that they are too many - it should include only [your commit](https://github.com/sunflowerit/l10n-netherlands/pull/1/commits/bbfa74390e17712c9b5da5076ee95148eea7e5ee).
5. Follow [this guide](http://wiki.websandbox.nl/Git#Removing_some_commits_from_a_branch) to get out of trouble.